app/server.js

```diff
@@ -99,16 +100,18 @@ module.exports = class Server extends OtherClasses {
 
```

Comment:
I made this alphabetical, since in the long run that will keep it easier to manage